test/unit/repositories/test_attachment.py

```diff
@@ -99,7 +99,7 @@ class GetDSL(AttachmentRepoDSL):
     _obtained_attachment_out: AttachmentOut
     _get_exception: pytest.ExceptionInfo
 
-    def mock_get(self, attachment_id: str, attachment_in_data: dict) -> None:
+    def mock_get(self, attachment_id: str, attachment_in_data: Optional[dict] = None) -> None:
```

Comment:
```suggestion
    def mock_get(self, attachment_id: str, attachment_in_data: Optional[dict]) -> None:
```
Very minor point, but we always specify None in the mock gets everywhere else in IMS/for images here, so I would prefer to keep them consistent. I believe the idea is that its more readable by seeing the actual value in the test.

Reply:
I see. I will revert these changes then cheers.